### DISCUSSION TITLE
Markdown PSL Highlight

### DISCUSSION THREAD

**Discussion by BautyGarcia** [prisma/prisma#18394] [Q&A] [answered]
I am willing to include some Pris,ma code in my markdown file for a school project, but when building I recieve that `Could not find the language 'prisma', did you forget to load/include a language module`. Here is an example of the markdown code block:

```prisma
datasource db {
	provider =  "postgresql"
	url =  "postgresql://user:password@localhost:5432/mydb?schema=public"
}

generator client {
	provider =  "prisma-client-js"
}

model User  {
	id Int @id @default(autoincrement()) 
	name String 
	email String @unique 
	password String
}
```

I was not able to find the npm package of whatever I need to be able to use prisma as a language on markdowns.

**Comment by ludralph** [comment, marked as answer]
Hi @BautyGarcia 👋 

The error message you received indicates that the code highlighting feature you're using in your markdown editor or build process doesn't recognize the Prisma language.

To enable syntax highlighting for Prisma code blocks in markdown, you can try adding a language definition for Prisma to your markdown editor's configuration or to your build process.

If you're using VS Code, you can install the Prisma VS Code extension which includes syntax highlighting for Prisma files.

do you have any more questions around this issue? If no, it would be really helpful if you could quickly **mark this Discussion as _answered_** to indicate that your issue has been resolved. 

In case you do have any outstanding questions, feel free to let us know and we'd be happy to help!

**Reply by ludralph** [reply]
Hi @BautyGarcia 

do you have any more questions around this issue? If no, it would be really helpful if you could quickly **mark this Discussion as _answered_** to indicate that your issue has been resolved. 

In case you do have any outstanding questions, feel free to let us know and we'd be happy to help!

**Reply by ludralph** [reply]
Hi @BautyGarcia 

You may have missed my previous comments. Do let us know if you have any other questions or you can mark this discussion as answered.
